test/e2e/run.bash

```diff
@@ -20,6 +21,11 @@ BATS_EXTRA_ARGS=""
 source "${E2E_DIR}/lib/defer.bash"
 trap run_deferred EXIT
 
+function download_known_hosts() {
+  mkdir -p "${FLUX_ROOT_DIR}/bin"
```

Comment:
I think the `bin` directory must either be renamed to `cache` or the `known_hosts` should be stored elsewhere.